src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -151,39 +150,48 @@ export const PaymentContent: FC<Props> = props => {
       <Collapse open={paymentMethod === "US_BANK_ACCOUNT"}>
         {getPaymentMethodInfo(paymentMethod)}
         <Spacer mb={2} />
-        <BankAccountPickerFragmentContainer
-          me={me}
-          order={order}
-          paymentMethod="US_BANK_ACCOUNT"
-          bankAccountHasInsufficientFunds={bankAccountHasInsufficientFunds}
-          onSetBankAccountHasInsufficientFunds={
-            onSetBankAccountHasInsufficientFunds
-          }
-          onSetIsSavingPayment={onSetIsSavingPayment}
-          onSetBalanceCheckComplete={onSetBalanceCheckComplete}
-          onSetSelectedBankAccountId={onSetSelectedBankAccountId}
-          bankAccountSelection={bankAccountSelection}
-          onSetBankAccountSelection={onSetBankAccountSelection}
-          clientSecret={clientSecret}
-          onSetClientSecret={setClientSecret}
-        />
+        {paymentMethod === "US_BANK_ACCOUNT" && (
```

Comment:
this is for a small performance improvement. 

when we have a `Collapse` component that is not open but has children ( `<Collapse open={false}>...children...</Collapse>`), the children are still mounted into the DOM but just not visible in the UI. 

this made sense between Collapses for CreditCard and ACH for example, because a user can click on a radio button to see CreditCard picker and then another radio button to see BankAccount picker for ACH. it's good that we have the content of those Collapses already mounted so that the user sees a swift transition when clicking radios. 

it doesn't make sense to have that between ACH and SEPA, because they are mutually exclusive. we wouldn't want to render `BankAccountPickerFragmentContainer` for ACH when SEPA is the only choice and user cannot select ACH at all.

Reply:
if you remove that change, and log `paymentMethod` in `BankAccountPicker`, and choose SEPA, you'll see an unnecessary render for ACH as well:

<img width="740" alt="Screenshot 2022-09-09 at 10 24 36" src="https://user-images.githubusercontent.com/42584148/189308886-b59e56b6-9886-4a0d-9582-e196608d88c6.png">